docs/operate/agent-config.mdx

```diff
@@ -29,21 +29,21 @@ Examples:
 
 * `{ "db": "/path/to/dir" }` can be set with `--db "/path/to/dir"`
 * `{ "chains": { "ethereum": { "name": "ethereum" } } }` (abbreviated as `chains.ethereum.name` or `chains.<chain_name>.name`) can be set with `--chains.ethereum.name ethereum`
-* `{ "chains": { "avalanche": { "connection": { "url": "https://some-url.com" } } } }` (abbreviated as `chains.avalanche.connection.url` or `chains.<chain_name>.connection.url`) can be set with `--chains.avalanche.connection.url "https://some-url.com"`
+* `{ "chains": { "avalanche": { "rpcUrls": [ { "http": "https://some-url.com" } ] } } }` (abbreviated as `chains.<chain_name>.rpcurls.0.http`) can be set with `--chains.avalanche.rpcurls.0.http "https://some-url.com"`
```

Comment:
Probably wouldn't be using this specific example since we don't want people to override RPC urls this way right now?

Reply:
Forgot to change it here, thanks